src/main/java/com/company/TweetApplication.java

```diff
@@ -1,23 +1,30 @@
 package com.company;
 
 import com.company.resources.TweetResources;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import io.dropwizard.Application;
 import io.dropwizard.Configuration;
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
 
-public class TweetApplication extends Application<Configuration> {
+public class TweetApplication extends Application<TwitterAppConfiguration> {
     public static void main(String[] args) throws Exception {
         new TweetApplication().run(args);
     }
 
     @Override
-    public void initialize(Bootstrap<Configuration> bootstrap) {
+    public void initialize(Bootstrap<TwitterAppConfiguration> bootstrap) {
         // nothing to do yet
     }
 
+
     @Override
-    public void run(Configuration configuration, Environment environment) {
-        environment.jersey().register(new TweetResources());
+    public void run(TwitterAppConfiguration configuration, Environment environment) {
+        final TweetResources resource = new TweetResources(configuration.getoAuthConsumerKey(),
```

Comment:
why are you passing all of the individual fields down vs passing the configuration object?